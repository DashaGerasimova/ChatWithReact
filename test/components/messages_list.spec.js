import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import toJson from 'enzyme-to-json';
import { shallow } from 'enzyme';

import List from '../../app/containers/chat/components/messages_list/index'

it('should render a todo item ', () => {
  const tree = toJson(shallow(<List messages={[]} />));
  expect(tree).toMatchSnapshot();
});
	
it('should match its snapshot with items', () => {
  const items = ['message1', 'message2', 'message3'];
  const tree = renderer.create(
    <List messages={items} />
  ).toJSON();

expect(tree).toMatchSnapshot();
});