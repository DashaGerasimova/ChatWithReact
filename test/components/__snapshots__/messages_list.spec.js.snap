// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match its snapshot with items 1`] = `
<ul>
  <li />
  <li />
  <li />
</ul>
`;

exports[`should render a todo item  1`] = `<ul />`;
